commands/mute/tempmute.js

```diff
@@ -133,21 +133,18 @@ function recordMuteInDB(message, toTempMute, lengthOfTime, reason, connection) {
     });
 }
 
-function recordUnmuteInDB(toTempMute, connection) {
+function recordUnmuteInDB(connection) {
     let now = new Date()
     let timestamp = dateFormat(now, "yyyy-mm-dd HH:MM:ss")
 
-    var sqlInfractions2 = `INSERT INTO infractions (timestamp, user, action, length_of_time, reason, valid, moderator) 
-    VALUES ('${timestamp}', '${toTempMute.id}', 'cc!tempmute', NULL, 'tempmute expired', true, 'automatic')`;
-
     var sqlModLog2 = `INSERT INTO mod_log (timestamp, moderator, action, length_of_time, reason) 
-    VALUES ('${timestamp}', 'automatic', 'cc!tempmute', NULL, 'tempmute expired')`;
+    VALUES ('${timestamp}', 'automatic', 'cc!unmute', NULL, 'tempmute expired')`;
```

Comment:
Right now, this only logs that there was an automatic unmute because tempmute expired, but it doesn't say who got unmuted.  I want to say that we will probably want to know whose tempmute expired since we could potentially mute someone for a week or more.  

With that in mind, it might be better to keep the `toTempMute` parameter in this function and then change the action to be `'cc!unmute <@toTempMute's user id>'`

Reply:
Done